runtime/server/build.gradle.kts

```
@@ -72,7 +72,11 @@ quarkus {
   )
 }
 
-tasks.register("run") { dependsOn("quarkusRun") }
+tasks.register("run") {
```

```suggestion
tasks.register("runServer") {
```
"run" is a bit too generic, because we have many things (cli or admin tool) that can be "run".

---

I agree, but my main intension of this PR was to make this tasks visible, cause there are scripts and manuals which mention "run" and it's not obvious from what place it appears if you're not so experienced in quarkus/gradle. I decided to just specify a "group" for those hidden tasks and make them easy to find. 

The changing taskName can be delivered in next PR.